src/main/java/by/exadel/internship/dto/user/TechUserDTO.java

```diff
@@ -0,0 +1,13 @@
+package by.exadel.internship.dto.user;
+import by.exadel.internship.dto.enums.Technology;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TechUserDTO extends UserDTO {
```

Comment:
As far as I remember we decided to remove this class.

Reply:
done